src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -50,10 +61,12 @@ export const AdvancedSearchPage = () => {
   const categoryShould = (params.get(ResultParam.CategoryShould)?.split(',') as PublicationInstanceType[] | null) ?? [];
   const topLevelOrganizationId = params.get(ResultParam.TopLevelOrganization);
   const unitId = params.get(ResultParam.Unit);
+  const excludeSubunits = params.get(ResultParam.ExcludeSubunits) === 'true' ? 'true' : null;
 
   const resultSearchQueryConfig: FetchResultsParams = {
     categoryShould,
     contributorName: params.get(ResultParam.ContributorName),
+    excludeSubunits: Boolean(excludeSubunits),
```

Comment:
Når "castes" `'true'` til `'true'`, for så å castes til `true`, som egentlig er overflødig.
```suggestion
    excludeSubunits: params.get(ResultParam.ExcludeSubunits) === 'true',
```